pkgs/top-level/python-packages.nix

```diff
@@ -6049,6 +6049,10 @@ self: super: with self; {
 
   nessclient = callPackage ../development/python-modules/nessclient { };
 
+  nest = toPythonModule(pkgs.nest.override { withPython = true; python3 = python; });
+
+  nest-mpi = toPythonModule(pkgs.nest-mpi.override { withPython = true; python3 = python; });
```

Comment:
We should only include one. How about nest but with mpi?

Reply:
Nice catch. Done in the new commit.